Libraries/src/Amazon.Lambda.RuntimeSupport/Bootstrap/UserCodeInit.cs

```diff
@@ -140,6 +140,9 @@ public static void LoadStringCultureInfo()
             }
         }
 
+#if NET8_0_OR_GREATER
+    [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCode("PreJitAssembly is not used for Native AOT")]
```

Comment:
Technically someone could want to trim their assembly and not use native AOT, although obviously native AOT is the big use case.